documentation/modules/ostack-app-cred-auth.adoc

```diff
@@ -72,6 +72,7 @@ EOF
 
 ** For authentication using the application credential name:
 +
+[source,yaml]
```

Comment:
this is just a missing definition for the codeblock... probably made no difference to anyone

Reply:
Good catch! As you say, better safe than sorry!